azure-pipelines/1esmain.yml

```diff
@@ -1,3 +1,8 @@
+parameters:
+- name: useAzureFederatedCredentials

```

Comment:
Can you add a comment here explaining what this variable does for someone who doesn't want to become a plumber to figure it out?

Reply:
Yes lol, done. Does it make sense now?